src/create.ts

```diff
@@ -61,13 +62,17 @@ export async function create(options: CreateOptions): Promise<Readable> {
 				},
 			},
 		},
-	})) as unknown as ReleaseManifest;
+	})) as SDK.Release;
 
 	if (!remoteRelease) {
 		throw new Error('Release not found.');
 	}
 
-	const bundle = new resourceBundle.WritableBundle<ReleaseManifest>({
+	if (remoteRelease.status !== 'success') {
+		throw new Error('Release should have a status that is equal to "success".');
```

Comment:
If you were a user of this library and met this error message, would you find it useful? What would you want the error to tell you?

Reply:
I updated the error message to include the constraint the user was hitting, explained in a less technical way.